### DISCUSSION TITLE
“No Smoke” and “Seatbelts” switches in A320neo

### DISCUSSION THREAD

**Discussion by Adi-18** [nguyenquyhy/Flight-Tracker-StreamDeck#143] [Q&A]
Thank you for that great tool. I have a question about using switches like “No Smoke” and “Seatbelts” in A320neo. I think I have to use:
CABIN_NO_SMOKING_ALERT_SWITCH_TOGGLE for “Toggle event” and
CABIN_NO_SMOKING_ALERT_SWITCH for “Feedack value”

CABIN_SEATBELTS_ALERT_SWITCH_TOGGLE for “Toggle event” and
CABIN_SEATBELTS_ALERT_SWITCH for “Feedack value”

But it those not works.
Do I something wrong or are those not supported? 🤔 
Thank you for feedback 😊

**Comment by nok3** [comment]
First, a very big thank you Mr. Nguyễn Quý Hy 

I have exactly the same problem, belt and not smoke, does not work. 
I have to write above the mention inactive pending.

I hope it is not frowned upon to put an image. 

<img src="https://images.flightsim.to/images/13/i_gQ9Bfm.jpg?width=1400&auto_optimize=small" />

**Reply by nguyenquyhy** [reply]
Wow did you build that profile?

**Reply by nok3** [reply]
Yes, I only copy the radio and the autopilot.

I retrieve a location on the bridge by putting the value of the shutters on the button. it is not possible to change the color of the value. so I change the color of the keys. if one day it is possible, to customize the colors

Sorry for the English translated by Google 

<https://drive.google.com/file/d/1mThBAxiYnP4s4Fsvr7c_pBmpfkmpZXhk/view" />

**Reply by imodude** [reply]
I've been trying out this profile - some great stuff in there - in particular, the 2-button throttle is awesome.

I've been having issues with the Com/Nav buttons tho - they seem fine in C172, but in A320 (possibly both versions, cant remember), the frequencies dont display on the buttons, and a long press doesnt reliably open up the num pad.

**Comment by nguyenquyhy** [comment]
Those events are in the TOGGLE_EVENTS list, so it will be sent to the sim. It's totally up to the aircraft dev to support that event or not.

If it's the default 320, I would suggest filing a ZenDesk ticket to Asobo. If it is the 32NX, I would suggest creating a ticket in their GitHub.

**Reply by nok3** [reply]
Thank you very much, and great job you are doing.

**Comment by phenDruck** [comment]
> Thank you for that great tool. I have a question about using switches like “No Smoke” and “Seatbelts” in A320neo. I think I have to use:
> 
> CABIN_NO_SMOKING_ALERT_SWITCH_TOGGLE for “Toggle event” and
> 
> CABIN_NO_SMOKING_ALERT_SWITCH for “Feedack value”
> 
> 
> 
> CABIN_SEATBELTS_ALERT_SWITCH_TOGGLE for “Toggle event” and
> 
> CABIN_SEATBELTS_ALERT_SWITCH for “Feedack value”
> 
> 
> 
> But it those not works.
> 
> Do I something wrong or are those not supported? 🤔 
> 
> Thank you for feedback 😊 

NOK i am so hopelessly lost yet I appreciate how powerful this utility is. By way of learning, would you be gracious and share your profile/setup so I could learn from it?  Thanks in advance

**Reply by nok3** [reply]
With pleasure.

https://fr.flightsim.to/file/13562/stream-deck-3x15-for-flight-simulator-flight-tracker-free-a320-for-the-time-being

**Reply by nguyenquyhy** [reply]
We have a discussion thread to share profile: https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/128

I have also shared my own profile (very simple one) in this discussion: https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/235

I am also working on a better way to share profile, but for now you can use the thread I shared above.
